.github/workflows/push.yml

```
@@ -713,7 +713,66 @@ jobs:
+        uses: actions/checkout@v4
+        with:
+          repository: island-is/feature-values
+          token: ${{ secrets.DIRTY_FIX_BOT_TOKEN }}
+          path: feature-values
+      - name: Cache for NodeJS dependencies - Infra
+        id: infra-node-modules
+        continue-on-error: true
+        uses: ./.github/actions/cache
+        with:
+          path: monorepo/infra/node_modules
+          key: ${{ needs.prepare.outputs.infra-node-modules-hash }}
```

actionlint (1.7.4)

property "infra-node-modules-hash" is not defined in object type {build_chunks: string; cache_keys: string; docker_tag: string; images: string; last_good_build_docker_tag: string; node-modules-hash: string; node_image_tag: string; test_chunks: string; unaffected: string}

(expression)